pkg/katautils/create.go

```diff
@@ -204,6 +204,16 @@ func CreateSandbox(ctx context.Context, vci vc.VC, ociSpec oci.CompatOCISpec, ru
 		return nil, vc.Process{}, err
 	}
 
+	defer func() {
+		// cleanup netns if kata creates it
+		ns := sandboxConfig.NetworkConfig
+		if err != nil && ns.NetNsCreated {
+			if ex := cleanupNetNS(ns.NetNSPath); ex != nil {
+				kataUtilsLogger.Warnf("Failed to cleanup netns %s", ns.NetNSPath)
```

Comment:
please use `WithField` in all the logs to avoid using format specifiers. for example 

```go
kataUtilsLogger.WithField("path", ns.NetNSPath).Warn("Failed to cleanup netns")
```

Reply:
Updated.